lib/hanami/cli/parser.rb

```diff
@@ -13,6 +13,7 @@ module Parser
       #
       # rubocop:disable Metrics/MethodLength
       def self.call(command, arguments, names)
+        original_arguments = arguments.dup
```

Comment:
we mutate arguments value, that's why I need to create a copy for error messages